packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js

```diff
@@ -156,19 +155,23 @@ exports.questions = {
 		validators: [new RequiredValidator()]
 	}),
 	representationUpload: new MultiFileUploadQuestion({
-		title: 'Upload representations from other parties',
-		question: 'Upload the representations',
-		fieldName: 'representation-upload',
-		validators: [new RequiredValidator('You must add your documents')]
+		title: '**************placeholder*********************',
+		question: '**************placeholder*********************',
+		fieldName: 'representation-upload'
+	}),
+	reportUpload: new MultiFileUploadQuestion({
```

Comment:
This section was a duplicate of the below and removed in the below pr earlier today, i think this is a merge artifact: https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/3055